src/main/java/chat/Ui.java

```diff
@@ -0,0 +1,119 @@
+     */
+    public void greet() {
+        System.out.println(greetingCat);
+        System.out.println("Mew! I'm Chat the Cat");
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Prints goodbye message from Chat the Cat.
+     */
+    public void goodbye() {
+        System.out.println("*** Goodbye *** " + goodByeCat);
+    }
+
+    /**
+     * Prints message from Chat the Cat to tell user that a task has been added successfully to list of tasks.
+     * 
+     * @param task Task that has been added to list of tasks.
+     * @param size Total number of tasks in the list of tasks.
+     */
+    public void printAddedSuccess(Task task, int size) { 
+        System.out.println(goCat);
+        System.out.println("Mew! I've added this task:");
+        System.out.println(task);
+        System.out.println(String.format("\n** Now you have %d tasks in the list **", size));
+    }
+
+    /**
+     * Prints message from Chat the Cat to tell user that a task has been deleted successfully from the list of tasks.
+     * @param task Task that has been deleted from list of tasks.
+     * @param size Total number of tasks in the list of tasks.
+     */
+    public void printDeleteSuccess(Task task, int size) {
+        System.out.println(goCat);
+        System.out.println("Mew! I've removed this task:");
+        System.out.println(task);
+        System.out.println(String.format("\n** Now you have %d tasks in the list **", size));
+    }
+
+    /**
+     * Prints message from Chat the Cat to tell user that a task has been marked successfully as done when completed.
+     * @param task Task Task that has been completed.
+     */
+    public void printWellDone(Task task) {
+        System.out.println(gdJobCat);
+        System.out.println("Mew! I've marked this task as done:");
+        System.out.println(task);
+        System.out.println("\n* Good job, you deserve a kit-kat *");
+    }
+
+    /**
+     * Retrieve next command from user to Chat the Cat.
+     * 
+     * @return Inputted command string from user to Chat the Cat.
+     */
+    public String readCommand() { 
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints error message when Chat has encountered an error when loading tasks from data on hard disk.
+     */
+    public void showLoadingError() { 
+        System.out.println(new ChatException("Error loading file"));
+    }
+
+    /**
+     * Prints error message when Chat has encountered an error.
+     * 
+     * @param e Error message from Chat to user.
+     */
+    public void showError(ChatException e) { 
+        System.out.println(errorCat + "\n" + e);
+    }
+
+    /**
+     * Displays tasks from list of tasks from TaskList object.
+     * 
+     * @param tasks TaskList object that contains a list of tasks.
+     */
+    public void list(TaskList tasks) {
+        int i = 0;
+        System.out.println(" * list *");
+        while (i < tasks.getTasks().size()) {
```

Comment:
i think its more standard to use the standard for loop in this scenario, even though they do the same thing... so that the code can be less surprising and more standard